cmd/influx/setup.go

```diff
@@ -2,13 +2,16 @@ package main
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"os"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/influxdata/influxdb"
 	platform "github.com/influxdata/influxdb"
```

Comment:
lets consolidate here, I'd much prefer to see us drop `platform`, but that's my $0.02

Reply:
and you are absolutely right